src/olympia/addons/tests/test_views.py

```diff
@@ -48,14 +48,7 @@
 from olympia.constants.browsers import CHROME
 from olympia.constants.categories import CATEGORIES, CATEGORIES_BY_ID
 from olympia.constants.licenses import LICENSE_GPL3
-from olympia.constants.promoted import (
```

Comment:
If we import a tuple we can't reference them by their name - they would just be a list of classes.  

We could import them as `amo.promoted.XXX` but that's rarely used - the convention everywhere has been to import them separately.  Changing it for just this file would break the convention and changing it everywhere is out of scope.